Experimental build of v0.6.3.dev1 using flang for win/py3k

brian-rose opened the pull request.

brian-rose committed:
Try again building for win/py36

brian-rose committed:
MNT: Re-rendered with conda-smithy 2.4.4

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 2.4.5

brian-rose committed:
Use the full fast test suite

brian-rose committed:
Merge branch 'master' of https://github.com/conda-forge/climlab-feedstock

brian-rose committed:
Use flang to build for win/py3k

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

brian-rose commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 2.4.5

brian-rose committed:
Skip build on for win/py2k.

Not sure why this is failing since it used to work fine, but win/py3k is higher priority.

brian-rose commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 2.4.5

brian-rose committed:
Only build for win/py36 -- for now at least

brian-rose commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 2.4.5

brian-rose committed:
Also exclude osx/py35 because tests fail with numpy 1.14 for some strange reason

brian-rose commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 2.4.5

brian-rose committed:
We lost win/py36 somehow. Try to render again.

brian-rose commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-smithy 2.4.5
